consensus/src/block_storage/block_store.rs

```diff
@@ -232,9 +256,10 @@ impl BlockStore {
                     panic!("[BlockStore] failed to insert quorum during build{:?}", e)
                 });
         }
-        counters::LAST_COMMITTED_ROUND.set(block_store.root().round() as i64);
+
+        counters::LAST_COMMITTED_ROUND.set(block_store.ordered_root().round() as i64);
         counters::LAST_COMMITTED_VERSION
-            .set(block_store.root().compute_result().num_leaves() as i64);
+            .set(block_store.ordered_root().compute_result().num_leaves() as i64);
```

Comment:
we probably should remove this, this is set to 0

Reply:
resolved